.github/workflows/markdown-link-checker.yml

```diff
@@ -1,14 +1,27 @@
 name: Check Markdown links
 
-on: push
+on: [pull_request]
 
 jobs:
   markdown-link-check:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - macos-latest
```

Comment:
Why it can't read with "ubuntu-latest"?

Reply:
I tried with "ubuntu-latest" but there was always an error.

Reply:
What error?

Reply:
Got it.